AdminClient/src/app/supplier/store/suppliers.effects.ts

```diff
@@ -0,0 +1,145 @@
+import { Injectable } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { Action, Store } from '@ngrx/store';
+import { of } from 'rxjs';
+import { delay, map, switchMap, take, tap } from 'rxjs/operators';
+import * as fromApp from '../../reducers/index';
+import { PopupDialogComponent } from '../../shared/components/popup-dialog/popup-dialog.component';
+import { PopupDialogDataModel } from '../../shared/model/popup-dialog-data.model';
+import { SupplierModel } from '../../shared/model/supplier/supplier.model';
+import { SupplierFormComponent } from '../supplier-form/supplier-form.component';
+import * as SupplierActions from './suppliers.actions';
+import * as fromSuppliers from './suppliers.reducer';
+
+@Injectable()
+export class SuppliersEffects {
+  deleteCustomer$ = createEffect(() => this.actions$.pipe(
```

Comment:
```suggestion
  deleteSupplier$ = createEffect(() => this.actions$.pipe(
```

and check referneces